### ISSUE TITLE
battleground AFK check is broken; penalizing players who have long queue times

### ISSUE THREAD

**Issue by nooban300** [slowtorta/turtlewow-bug-tracker#3888]
**Please, provide a clear description what the bug is:**
The AFK status does not clear when I move the mouse and click the ENTER BATTLEGROUND invitation.  The result of this erroneous AFK status is that as soon as you enter the battleground, you are immediately kicked out and given a 15 minute debuff wasting potentially an hour or more of waiting.  This can be fixed in at least two ways: 1) Do not allow characters marked as AFK to join battlegrounds (this functionality already exists in ghost form).  2) Clear the AFK status when moving the mouse and/or interacting with the battleground invitation.  **Thank you for all your work on Turtle!!**
**Steps to reproduce the behavior with as much detail as possible:**

1. Be in a queue for a battleground long enough to be marked as AFK (5 minutes).
2. Use a mouse to Interact with the battleground invitation and click ENTER BATTLEGROUND.
3. You will enter the battleground, immediately you will be kicked out and given a 15 minute debuff despite being at the keyboard, moving the mouse and interacting with the battleground invitation.

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=
Item: https://database.turtle-wow.org/?item=
Quest: https://database.turtle-wow.org/?quest=
Spell: https://database.turtle-wow.org/?spell=
Object: https://database.turtle-wow.org/?object=

**Expected behavior. Describe how it should work:**
1) Do not allow characters marked as AFK to join battlegrounds (this functionality already exists while in ghost form; cannot join battlegrounds while dead).
2) Clear the AFK status when moving the mouse and/or interacting with the battleground invitation.

**Comment by balakethelock** [comment]
Can this be avoided by moving to remove afk or manually removing it with the chat command?

**Comment by nooban300** [comment]
Yes.  It could also be avoided by not participating in the battlegrounds, or the entire game for that matter.  

The point is that it's a pitfall for players who spend a lot of time waiting in the battleground queue.  I think most would agree that this functionality isn't congruent with the purpose of the AFK system unless the point is to punish people who fail to remember to first shake your character 'awake' before interacting with the battleground invitation.  

Interacting with the battleground invitation should be a clear and obvious indication that the player is present.
